### PR TITLE
[Blueprint] Update version number and changelog for a new version

### PR DESCRIPTION
**Pull request by filizt**
## Checklist

- [x] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [ ] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

**Comment by filizt** [comment]
@VeryEarly @isra-fel This is the changelog and version update for Blueprints module. Could you publish a new version of the Blueprints module? This is a preview module and it goes out on demand.

File: src/Blueprint/Blueprint/Az.Blueprint.psd1 [hunk 1]
```diff
@@ -12,7 +12,7 @@
 # RootModule = ''
 
 # Version number of this module.
-ModuleVersion = '0.2.7'
+ModuleVersion = '0.2.8'
```
Comment (VeryEarly) [inline src/Blueprint/Blueprint/Az.Blueprint.psd1]: do not upgrade version

File: src/Blueprint/Blueprint/ChangeLog.md [hunk 2]
```diff
@@ -19,6 +19,9 @@
 -->
 ## Upcoming Release
 
+## Version 0.2.8
```
Comment (VeryEarly) [inline src/Blueprint/Blueprint/ChangeLog.md]: put change log under upcoming release

Comment (filizt) [reply src/Blueprint/Blueprint/ChangeLog.md]: @VeryEarly done

File: src/Blueprint/Blueprint/Az.Blueprint.psd1 (hunk 1 above)
Comment (filizt) [reply src/Blueprint/Blueprint/Az.Blueprint.psd1]: @VeryEarly done.

**Pull request merged**